CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+## 1.1.0 / 2016-09-01
+
+* [CHANGE] Kubernetes SD: Drop `__meta_kubernetes_role`.
+* [FEATURE] Add `quantile` and `quantile_over_time`.
+* [FEATURE] Add `stddev_over_time` and `stdvar_over_time`.
+* [FEATURE] Add various time and date functions.
+* [FEATURE] Added `toUpper` and `toLower` formatting to templates.
+* [FEATURE] Allow relabeling of alerts.
+* [FEATURE] Allow URLs in targets defined via a JSON file.
+* [FEATURE] Add idelta function.
+* [FEATURE] 'Remove graph' button on the /graph page.
+* [FEATURE] Kubernetes SD: Add node name and host IP to pod discovery.
+* [FEATURE] New remote storage write path. EXPERIMENTAL!
+* [ENHANCEMENT] Improve time-series index lookups.
+* [ENHANCEMENT] Forbid invalid relabel configurations.
+* [ENHANCEMENT] Improved various tests.
+* [ENHANCEMENT] Add crash recovery metric 'started_dirty'.
+* [ENHANCEMENT] Fix (and simplify) populating series iterators.
+* [ENHANCEMENT] Add job link on target page.
+* [ENHANCEMENT] Allow query parameters for the /graph endpoint.
+* [ENHANCEMENT] Message on empty Alerts page.
+* [ENHANCEMENT] Various internal code refactorings and clean-ups.
+* [ENHANCEMENT] Various improvements in the build system.
+* [BUGFIX] Catch errors when unmarshalling delta/doubleDelta encoded chunks.
+* [BUGFIX] Fix data race in lexer and lexer test.
+* [BUGFIX] Trim stray whitespace from bearer token file.
+* [BUGFIX] Avoid divide-by-zero panic on query_range?step=0.
+* [BUGFIX] Detect invalid rule files at startup.
```

Comment:
This might actually qualify as a `[CHANGE]`. It was possible to reference invalid rule files and Prometheus would just always ignore that. Now the server won't start anymore.